bin/micro.js

```diff
@@ -82,6 +82,15 @@ if (!existsSync(file)) {
   process.exit(1)
 }
 
+const { port, host } = flags
+if (isNaN(port) || (!isNaN(port) && (port < 1 || port >= 2 ** 16))) {
```

Comment:
You should prefer `Number.isNaN` over `isNaN`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN

Reply:
The `2 ** 16` statement is causing trouble for me on Node 8.9.1.
```
if (isNaN(port) || (!isNaN(port) && (port < 1 || port >= 2 ** 16))) {
                                                            ^
SyntaxError: Unexpected token *
```
Any reason to use this over `Math.pow`?

Reply:
shouldn't we specify an `engines` field in the package.json so we at least get a npm warning when installing ?

Reply:
Nothing against `Math.pow`, just sugar syntax there. And since micro expects the usage of `async/await` i thought something that was available earlier wouldn't hurt.

https://github.com/zeit/micro/blob/2408dd57a2c2e4072563d7be0967652deeddb0bd/lib/handler.js#L23

Reply:
for some reason the error message was not displayed in my case with node v 6.2.3

upgraded to 8.7 and 👌

<img width="703" alt="capture d ecran 2017-11-14 a 00 54 03" src="https://user-images.githubusercontent.com/124937/32755670-ac83c3c4-c8d6-11e7-9675-2afd22416cac.png">